partner_search_alias/models/res_partner.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2024 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    search_alias = fields.Char(help="Enter the name that is also used for name search.")
+
+    @api.model
+    def _name_search(
+        self, name="", args=None, operator="ilike", limit=100, name_get_uid=None
+    ):
+        args = list(args or [])
+        if name:
+            args += ["|", ("search_alias", operator, name)]
+        return super()._name_search(name, args, operator, limit, name_get_uid)
```

Comment:
@AungKoKoLin1997 Please try this.
```suggestion
    @property
    def _rec_names_search(self):
        return list(set(super()._rec_names_search + ["search_alias"]))
```

Reply:
It works. Thank you.